package.json

```diff
@@ -77,7 +77,6 @@
     "react-sparklines": "1.7.0",
     "reflux": "0.4.1",
     "scroll-to-element": "^2.0.0",
-    "select2": "3.5.1",
```

Comment:
👍

Reply:
i think i recall a css file that we'd need to manually remove as well

Reply:
Yeah it's removed (did I forget to push some updates?) https://github.com/getsentry/sentry/pull/8743/files#diff-ac24416e63ab1dd5bbc404fa65215557L7

Reply:
You're good, i missed it in the diff